### PR TITLE
Add tests for ObjectiveBound

### PR DESCRIPTION
**Pull request by odow**
Ref #442

### CODE REVIEW

File: src/Test/UnitTests/solve.jl [hunk 1]
```diff
@@ -0,0 +1,81 @@
+"""
+    solve_objbound_edge_cases(model::MOI.ModelLike, config::TestConfig)
+
+Test a variety of edge cases related to the ObjectiveBound attribute.
+"""
+function solve_objbound_edge_cases(model::MOI.ModelLike, config::TestConfig)
+    @testset "Min IP with constant" begin
+        MOI.empty!(model)
+        @test MOI.isempty(model)
+        MOIU.loadfromstring!(model,"""
```
Comment (IainNZ) [inline src/Test/UnitTests/solve.jl]: Nittiest of nits: space before `"""`

File: src/Test/UnitTests/solve.jl [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            variables: x
+            minobjective: 2.0x + -1.0
+            c1: x >= 1.5
+            c2: x in Integer()
+        """)
+        x = MOI.get(model, MOI.VariableIndex, "x")
+        test_model_solution(model, config;
+            objective_value   = 3.0,
```
Comment (IainNZ) [inline src/Test/UnitTests/solve.jl]: Uncessary white space?

File: src/Test/UnitTests/solve.jl (hunk 1 above)
Comment (IainNZ) [reply src/Test/UnitTests/solve.jl]: Repeated below.

File: src/Test/UnitTests/solve.jl (hunk 2 above)
Comment (IainNZ) [reply src/Test/UnitTests/solve.jl]: Repeated below.
